packages/deployer/README.md

```diff
@@ -0,0 +1,117 @@
+# @airnode/deployer
+
+> Deployment tools for Airnode
+
+*The recommended way to deploy Airnode is by using the deployment Docker image.
+This package simply implements the scripts used by that image and is not meant to be used directly by the end user.*
+
+*If you will be using the provided scripts locally, run `export-env.sh` to export your AWS credentials as environment variables first.*
+
+## Setup
+
+- Download the [Terraform v0.13.* binary](https://www.terraform.io/downloads.html) and move it to your `PATH`
+- Get an *Access Key ID* and *Secret Access Key* from AWS and insert into `.env` (refer to `.env.example`)
+- Install Serverless Framework globally
+```sh
+npm install -g serverless
+```
+
+## Basics
+
+Airnode is deployed in two steps:
+
+1. Deploy the mnemonic at AWS SSM at the `region` that the serverless functions will be deployed at using Terraform
+
+1. Deploy the serverless functions at the target `region` using Serverless Framework
+
+The mnemonic deployment needs to be performed only when deploying for the first time on a region.
+For the following redeployments, mnemonic deployment must not be repeated.
+
+## Regular user flow
+
+1. Use the `deploy-first-time` for your first deployment.
+Note down the displayed mnemonic.
```

Comment:
They can deploy their own mnemonic using [`deploy-mnemonic`](https://github.com/api3dao/airnode/blob/aws-deployment/Docker.md#deploy-mnemonic), use the corresponding `providerIdShort` in their `config.json` and use `redeploy`. I didn't bother optimizing this flow because it's for pros only.

>ensuring they have it written down, by forcing them to enter it again

and probably allow to disable that with a flag. I think having them type in the mnemonic is a security issue with potential keyloggers. We should probably ask a single word at random, like "what was your Xth word?". Let me know what you think.

Reply:
> type in the mnemonic is a security issue with potential keyloggers

Good point. Much better to **not** have them enter the full mnemonic. I like having them enter a random Xth word 👍🏻

Reply:
Created an issue https://github.com/api3dao/airnode/issues/143